### ISSUE TITLE
Commits missing on webpage

### ISSUE THREAD

**Issue by cdeil** [spacetelescope/asv#171]
I have a case where the webpage is not showing all commits for which there is benchmark info.

There should be two commits:
```
$ ls -1 results/ChristophMacBook/
1b40df35-py2.7-Cython-numpy.json
1b40df35-py3.4-Cython-numpy.json
7060f907-py2.7-Cython-numpy.json
7060f907-py3.4-Cython-numpy.json
machine.json
```

But I only see one: `1b40df35^!`
And the overview page doesn't contain anything (probably needs at least two points for a line to show up).

@mdboom Can you reproduce this issue by only benchmarking two commits?

**Comment by mdboom** [comment]
I suspect #168 fixes this (merged only minutes ago).  Can you confirm it works for you, and close this if so?

**Comment by cdeil** [comment]
Yes, now it looks OK.

One more minor thing ... I have benchmarks that failed for all commits (need do have a look why) and that gives:
```
asv.js:729 Uncaught TypeError: Cannot read property '2' of undefined
```
If you don't think it's worth fixing, please just close this issue.

**Comment by mdboom** [comment]
I think the js error is fixed by #163, also just merged.  Let me know if not.

**Comment by cdeil** [comment]
Yes, it's fixed. Thanks!
